systemtests/tests/pruning/testrunner

```diff
@@ -149,87 +166,136 @@ prune files yes
 wait
 restore jobid=11
 no
-
 messages
 quit
 END_OF_DATA
 
 run_bconsole
 
-check_for_zombie_jobs storage=File
+# Purging failed jobs
 
-check_two_logs
-
-if ! grep "Purging the following 2 JobIds: 1,2" "$tmp"/log2.out; then
-  echo "'Purging the following JobIds: 1,2' not found in $tmp/log2.out" >&2
-  grep 'Purging the' "$tmp"/log2.out >&2 || :
-  estat=1
-fi
-
-if ! grep -F 'Volume "TestVolume001" contains no jobs after pruning.' $tmp/log2.out; then
-  echo "'Volume \"TestVolume001\" contains no jobs after pruning.' not found in $tmp/log2.out" >&2
-  grep 'Volume.*contains no jobs after pruning.' $tmp/log2.out >&2 || :
-  estat=1
-fi
-
-## prune all volumes checks
-
-if ! grep "Purging the following 2 JobIds: 4,5" "$tmp"/log3.out; then
-  echo "'Purging the following JobIds: 4,5' not found in $tmp/log3.out" >&2
-  grep 'Purging the' "$tmp"/log3.out >&2 || :
-  estat=1
-fi
-
-if ! grep -F 'Volume "TestVolume002" contains no jobs after pruning.' $tmp/log3.out; then
-  echo "'Volume \"TestVolume002\" contains no jobs after pruning.' not found in $tmp/log3.out" >&2
-  grep 'Volume.*contains no jobs after pruning.' $tmp/log3.out >&2 || :
-  estat=1
-fi
-
-if ! grep "Purging the following 2 JobIds: 6,7" "$tmp"/log3.out; then
-  echo "'Purging the following JobIds: 6,7' not found in $tmp/log3.out" >&2
-  grep 'Purging the' "$tmp"/log3.out >&2 || :
-  estat=1
-fi
-
-if ! grep -F 'Volume "TestVolume003" contains no jobs after pruning.' $tmp/log3.out; then
-  echo "'Volume \"TestVolume003\" contains no jobs after pruning.' not found in $tmp/log3.out" >&2
-  grep 'Volume.*contains no jobs after pruning.' $tmp/log3.out >&2 || :
-  estat=1
-fi
+purging_failed_jobs="$tmp"/purging_failed_jobs.out
 
-# Prune all volumes of specific pool
+cat <<END_OF_DATA >"$tmp/bconcmds"
+@$out $purging_failed_jobs
+run job=$failingjob_name level=Full yes
+run job=$failingjob_name level=Full yes
+run job=$failingjob_name level=Full yes
+wait
+messages
+purge jobs client jobstatus=E yes
```

Comment:
```suggestion
purge jobs client=bareos-fd jobstatus=E yes
```
Cosmetical change, but without I've been confused at first (if only only client is configured, it gets autoselected).